### ISSUE TITLE
Using dark theme text is shown in grey

### ISSUE THREAD

**Issue by norcino** [unindented/storybook-addon-rtl#23]
I use @storybook/theming ad dev dependecy and in manager.js
```
addons.setConfig({
  theme: themes.dark
});

```

doing so the text "Left-to-right" is still shown in dark grey making it impossible to read.

Is it possible to customise it?
Can I otherwise create a global button at the top?

**Comment by literalpie** [comment]
Hi, I don't think this has ever been tested with theming in mind, or has any way to customize the text color. That seems like a reasonable thing to ask for, so we should be able to add support for that. (PRs welcome :-))

I'm not really sure what you mean by "a global button at the top".

**Comment by norcino** [comment]
> Hi, I don't think this has ever been tested with theming in mind, or has any way to customize the text color. That seems like a reasonable thing to ask for, so we should be able to add support for that. (PRs welcome :-))

I am not an expert front end developer but i can try to figure a way to handle it.

> I'm not really sure what you mean by "a global button at the top".

Fair :), you can add global buttons in the toolbar at the top of the page, there you can add custom buttons to switch language, for example, and I would like to have access there to the LTR feature, this has been very useful already!

**Comment by literalpie** [comment]
I see what you're saying. I think that could be nice. 
I don't think I would want to make that change by default because it is a major change for people who currently expect the toggle in the panel. However I think it would be interesting to at least prototype, and possibly have an opt-in way of changing where the toggle is.

**Comment by norcino** [comment]
Yes the option to have it is a good idea, I rather activate the toolbar rather than the panel, because I find the behaviour clashing a bit with the idea that Panel's context is the story (generally), toolbar is global.
And when I enable RTL for a story, that setting will remain active until I manually switch it, even changing stories.
Just saying, I am no expert, it has been just around a month that I got started with storybook.

**Comment by MXTcomunica** [comment]
In my opinion having this placed in the "top right" global tools as "global option" affecting all stories would be helpful in improving the overall UX of this feature.

+1
